autogpt_platform/backend/backend/api/features/integrations/router_test.py

```
@@ -856,6 +856,58 @@ def test_owned_webhook_pings(self, test_user_id):
+            patch.dict(
+                "backend.api.features.integrations.router.DEVICE_HANDLERS_BY_NAME",
+                {"stripe_link": MagicMock()},
+                clear=False,
+            ),
+            patch.dict(
+                "backend.api.features.integrations.router.HANDLERS_BY_NAME",
+                {},
+                clear=True,
+            ),
+        ):
+            with pytest.raises(fastapi.HTTPException) as exc:
```

Ruff (0.16.1)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)